### DISCUSSION TITLE
Enable Authentication on Metamask with My Enterprise Credentials

### DISCUSSION THREAD

**Discussion by enlander** [MetaMask/snaps#849] [Ideas]
Enable authentication, privacy, and consent in ways that keep users in control.

I would like to add to metamask the capability to use my user credentials from AWS Cognito or Azure Active Directory. This can help to unlock enterprise adoption of Web3.  

Many Thanks

**Comment by ritave** [comment]
@enlander Thank you for writing. Could you expand on your idea? What would like to authenticate with your cloud credentials inside of MetaMask?
